xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js

```diff
@@ -583,6 +583,10 @@ define('xwiki-livedata', [
       return this.fetchEntries()
         .then(data => {
           this.data.data = Object.freeze(data);
+          let self = this;
+          Vue.nextTick(function () {
+            self.triggerEvent('entriesUpdated', {});
+          });
```

Comment:
An alternative is to use an arrow function expression instead of an anonymous function. It's shorter to write. In addition, arrow function does not have their own binding so saving `this` into `self` (which is would call `that` out of habit) is not required.
```suggestion
          Vue.nextTick(() => this.triggerEvent('entriesUpdated', {}));
```